README.md

```diff
@@ -12,84 +12,41 @@ Privacy Badger is a project of the Electronic Frontier Foundation.
 
 ## Developing
 
-### Load the extension from source code
+We're glad you want to help! We have a **[handy guide for
+contributors](/doc/CONTRIBUTING.md)**. It can hopefully point you to helpful
+resources if you have any questions (please let us know if it doesn't). To get
+started with most things, you'll need to [install Privacy Badger from
+source](#install-from-source)  But to hack on Privacy Badger you need to install
+it from source first, so we'll start there.
+
+### Install from source
+
+To hack on privacy badger you need to install it from source.
 
 In Chrome, visit `chrome://extensions`, enable "Developer mode", click "Load unpacked extension..." and select the [`src`](src/) subdirectory inside your copy of the Privacy Badger source code.
 
 In Firefox, visit `about:debugging`, click "Load Temporary Add-on" and select the `src` subdirectory.
 
+To test it out by running the unit tests. On chrome you can run them by clicking [here](chrome-extension://mekoeccjihallkjjjkcfpogljdnekmkd/tests/index.html), for Firefox follow [these instructions](/doc/tests.md#unit-tests).
```

Comment:
My dev extension ID is something different. They seem to vary on folder name, maybe other stuff.